packages/@tinacms/toolkit/src/hooks/use-local-storage.ts

```diff
@@ -16,8 +16,11 @@ import * as React from 'react'
 export function useLocalStorage(key, initialValue) {
   const [storedValue, setStoredValue] = React.useState(initialValue)
   React.useEffect(() => {
-    const valueFromStorage = window.localStorage.getItem(key)
-    if (valueFromStorage != null) {
+    // localStorage may be intentionally disabled for site visitors, 
+    // in that case Tina can't be used so just return false
+    const isEdit = window.localStorage && window.localStorage.getItem(LOCALSTORAGEKEY)
```

Comment:
is this supposed to be `window.localStorage.getItem(key)`?

Reply:
Oops, I ended up with an unfortunate copy-paste of the previous suggestion. This should be sorted now 🤕